tests/thermal_energy_density_error_message.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env perl
```

Comment:
please rename this file to `thermal_energy_density_error_message.pl` since it is a perl script. Also add a comment at the top explaining what this script does (it seems like you need to replace some output part, but why?)

Reply:
Oke, I will add a comment explaining what the script does. 

The normal output contains the whole path starting at root, which will be different for every testing machine. The other lines are just taken from the default script which is used for every test output. The tester succeeds locally for me, but I have to find out why it doesn't work with the online testers.

I could change the extension to pl, but then we would need to change it for all scripts (since they all use the perl) and change the `CMakeLists.txt` in the text directory:

https://github.com/geodynamics/aspect/blob/9faf7ec832a1ab002b348f65e08b275e1ec2bb95/tests/CMakeLists.txt#L245-L252